pages/app-layout/with-stacked-notifications-and-table.page.tsx

```diff
@@ -0,0 +1,155 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
The recently added page `app-layout/with-stacked-notifications-and-no-paddings` was deleted in favor of this one, which is more complete and makes it redundant. No tests had been written for the former yet.